codestodians/food-basket-front-end
# [Fix]: Fix CreateUser query

#### What does this PR do?
- Adjusts the possible query options on the `createUser` mutation due to changes on the backend

#### Description of Task to be completed?
- Adjust query options on the `createUser` mutation.
- Adjust query options in the mock client
- Update select apollo dependencies, refactor client based on upgrade
- Update snapshots

#### How should this be manually tested?
- Check out into the `feature/fix-create-user-query-options`.
- Run `yarn` or `yarn install`
- Run the app with `yarn start`
- It should work as before with no errors on the browser console.

#### What are the relevant pivotal tracker stories?
[#158291479](https://www.pivotaltracker.com/story/show/158291479)
